### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by alexblnn**
Adding two unit tests relevant to PR #20 (sorry for not adding this at the same time)

### CODE REVIEW

File: sanssouci/post_hoc_bounds.py [hunk 1]
```diff
@@ -118,8 +118,10 @@ def min_tdp(p_values, thr):
         confidence bounds on false positives using reference families.
         Annals of Statistics, 48(3), 1281-1303.
     """
-
-    return min_tp(p_values, thr) / len(p_values)
+    if len(p_values) == 0:
+        return 0
```
Comment (bthirion) [inline sanssouci/post_hoc_bounds.py:122]: Do we have a test for that case ?

File: sanssouci/tests/test_lambda_calibration.py [hunk 2]
```diff
@@ -33,6 +34,19 @@ def test_get_perm_p():
...
+    rng = np.random.RandomState(42)
+    n = 20
+    p = 50
+    X = rng.randn(n, p)
+    B = 100
+    pvals = get_permuted_p_values_one_sample(X, B=B)
+    assert pvals.shape == (B, p)
+    assert pvals.min() > 1.e-7
+    assert pvals.max() <= 1
+    assert np.sum(pvals < .1) < B * p * .12
```
Comment (bthirion) [inline sanssouci/tests/test_lambda_calibration.py:47]: If pvals are supposed to be sorted, you need to check that they are.

File: sanssouci/post_hoc_bounds.py (hunk 1 above)
Comment (alexblnn) [reply sanssouci/post_hoc_bounds.py:122]: Yes, see [https://github.com/alexblnn/sanssouci.python/blob/one_sample/sanssouci/tests/test_post_hoc_bounds.py#L78-L81](url)

Comment (alexblnn) [reply sanssouci/post_hoc_bounds.py:122]: Yes, see https://github.com/alexblnn/sanssouci.python/blob/one_sample/sanssouci/tests/test_post_hoc_bounds.py#L78-L81

File: sanssouci/tests/test_lambda_calibration.py (hunk 2 above)
Comment (alexblnn) [reply sanssouci/tests/test_lambda_calibration.py:48]: Done, see https://github.com/alexblnn/sanssouci.python/blob/one_sample/sanssouci/tests/test_lambda_calibration.py#L45 (maybe there is a smarter way to do this...)

**Review by bthirion — APPROVED** [review]
LGTM, thx.

**Comment by pneuvial** [comment]
Thanks!

**Pull request merged**